File: Challenge4/Python/Challenge4.py
def palindromic():	
	highestPalindromic = 0
	for x in range(0,1000):
		for y in range(1,1000):
			totalNumber = x*y
			totalNumberReversed = int(str(totalNumber)[::-1])
			if totalNumber == totalNumberReversed and highestPalindromic < totalNumber:
							highestPalindromic = totalNumber		
	return highestPalindromic
print (palindromic())

File: Challenge4/Ruby/Challenge4.rb
def palindromic
	highest_palindromic = 0
		for i in 1..999
			for j in 1..999
				
				total_number = i*j
				string_total_number = total_number.to_s
				reversed_string = string_total_number.reverse	
						
						if (string_total_number== reversed_string && highest_palindromic < total_number)
							highest_palindromic = total_number
						end
			end 
		end
	return highest_palindromic	 
end

puts palindromic